sdks/python/apache_beam/examples/dataframe/flight_delays_it_test.py

```diff
@@ -125,7 +125,8 @@ def read_csv(path):
       expected_df = expected_df.sort_values('airline').reset_index(drop=True)
 
       try:
-        pd.testing.assert_frame_equal(result_df, expected_df)
+        pd.testing.assert_frame_equal(
+            result_df, expected_df, check_dtype=False, atol=0.5)
```

Comment:
I'd like to better understand what went wrong with pandas 1.4.0 before making this change.

Reply:
For one I don't think check_dtype=False is necessary, secondly atol=0.5 is _very_ large. The default is 1e-8.

This is masking a difference of ~0.3 for expected result 23.823529 (~1%). Which I don't think we should just gloss over.

It would be preferable to skip this test and file a jira blocking 2.37.0

Reply:
Created Jira to track this https://issues.apache.org/jira/browse/BEAM-13867

Reply:
I put up a fix for this here: https://github.com/apache/beam/pull/16801

It turns out there was a slight change in the behavior of `nlargest` that affected this test: https://github.com/pandas-dev/pandas/issues/28984